lib/client/api.go

```diff
@@ -960,35 +952,42 @@ func (tc *TeleportClient) Logout() error {
 // Login logs the user into a Teleport cluster by talking to a Teleport proxy.
 // If successful, saves the received session keys into the local keystore for future use.
 func (tc *TeleportClient) Login() (*CertAuthMethod, error) {
+	httpsProxyHostPort := tc.Config.ProxyWebHostPort()
```

Comment:
I think this is good actualy because there are TWO "proxyhostport" values, one is HTTPS and another is SSH, so it makes it clearer which proxy port we're using here